factom_did/client/updater.py

```diff
@@ -320,7 +332,7 @@ def export_entry_data(self):
         if add_dict:
             entry_content_dict["add"] = add_dict
 
-        entry_content = json.dumps(entry_content_dict).replace(" ", "")
+        entry_content = json.dumps(entry_content_dict)
```

Comment:
I'm not sure we should be dropping this, as by default the conversion to string will introduce white space in the entries. However, it's better to use [this](https://kite.com/python/examples/336/json-encode-a-%60dict%60-to-a-json-string-without-spaces-in-separators) instead of the replace here.